server/controllers/assess/submittedApplicationsController.ts

```diff
@@ -41,10 +42,13 @@ export default class SubmittedApplicationsController {
 
       const status = application.statusUpdates.length ? application.statusUpdates[0].label : 'Received'
 
+      const showTimelineDescription = config.flags.notesDisabled === 'false'
```

Comment:
as the `Notes` have already been filtered out in the `utils` is this just hiding the `description` for the status update details, as below? But they aren't behind a feature flag atm right, was there a UCD requirement for them to be hidden?
![Screenshot 2024-02-22 at 14 58 01](https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui/assets/16647486/bc09137a-80e2-4fd0-94e4-3ae2df3b8bf5)